include/extractor/scripting_environment_lua.hpp

```diff
@@ -68,7 +68,14 @@ class LuaScriptingEnvironment final : public ScriptingEnvironment
                     tbb::concurrent_vector<boost::optional<InputRestrictionContainer>>
                         &resulting_restrictions) override;
 
+    void setExtractionContainers(ExtractionContainers &extractor) override;
+
+    // FIXME This is static because I don't know how to access to this in lua nodeRef class binding.
+    osmium::Location getLocation(osmium::NodeRef &nref);
```

Comment:
I think this is because, luabind try to call getLocation function with a NodeRef instance. Am I Right ?
Is that the way you asked me to do it?